mantidproject/mantid
# Enable deprecation warnings in python for development

As pointed out in [this blog post](https://metarabbit.wordpress.com/2014/05/27/protip-enable-deprecation-warnings-in-python/) and [documented by python](https://docs.python.org/2/using/cmdline.html#cmdoption-w), deprecation warnings are turned off by default. This turns them back on when using the launch scripts on linux for developers.

**To test:**

Look at the new launch scripts (development and install versions) and see the difference.

*There is no associated issue.*

*This does not require release notes* because it will only be seen by developers.

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/IndividualTicketTesting/)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
